torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -13,6 +13,12 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+using steady_clock_time_point =
+    std::chrono::time_point<std::chrono::steady_clock>;
+
+constexpr steady_clock_time_point kInfiniteTimeoutTimePoint =
```

Comment:
why do you need this in the .h?

Reply:
@jjlilley Wanted to put it in the class declaration.